src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java

```diff
@@ -48,37 +49,37 @@
             new LiteralText("No game is open in this world!")
     );
 
-    public static final DynamicCommandExceptionType MALFORMED_CONFIG = new DynamicCommandExceptionType(error -> {
-        return new TranslatableText("Malformed config: %s", error);
-    });
+    public static final DynamicCommandExceptionType MALFORMED_CONFIG = new DynamicCommandExceptionType(error ->
+            new TranslatableText("Malformed config: %s", error)
+    );
 
     // @formatter:off
     public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(
             literal("game")
                 .then(literal("open")
-                    .requires(source -> source.hasPermissionLevel(2))
-                    .then(GameConfigArgument.argument("game_config")
+                        .requires(Permissions.require("plasmid.command.game.open"))
```

Comment:
Change to Permissions.require("plasmid.command.game_channel", 2) which would make 2 a default in case the server isn't using a permission mamanger.